backend/src/main/java/io/github/k_gregory/registry/service/impl/EnforcementServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package io.github.k_gregory.registry.service.impl;
+
+import io.github.k_gregory.registry.model.Enforcement;
+import io.github.k_gregory.registry.repository.EnforcementRepository;
+import io.github.k_gregory.registry.service.EnforcementService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EnforcementServiceImpl implements EnforcementService {
```

Comment:
Do we need to register this interface-implementation somewhere?

Reply:
No, Spring Boot scans package for implementations automatically